generators/app/index.js

```diff
@@ -1,349 +1,57 @@
-      this.destinationPath('src/app/components/Routes/index.js'),
-      {}
-    );
-    this.fs.copy(
-      this.templatePath('src/app/components/Routes/styles.js'),
-      this.destinationPath('src/app/components/Routes/styles.js'),
-      {}
-    );
-    this.fs.copy(
-      this.templatePath('src/app/index.js'),
-      this.destinationPath('src/app/index.js'),
-      {}
-    );
-    this.fs.copy(
-      this.templatePath('src/config/api.js'),
-      this.destinationPath('src/config/api.js'),
-      {}
-    );
-    this.fs.copy(
-      this.templatePath('src/config/i18n.js'),
-      this.destinationPath('src/config/i18n.js'),
-      {}
-    );
-    this.fs.copy(
-      this.templatePath('src/redux'),
-      this.destinationPath('src/redux'),
-      {}
-    );
-    this.fs.copy(
-      this.templatePath('src/redux/store.js'),
-      this.destinationPath('src/redux/store.js'),
-      {}
-    );
-    this.fs.copy(
-      this.templatePath('src/redux/Auth/actions.js'),
-      this.destinationPath('src/redux/Auth/actions.js'),
-      {}
-    );
-    this.fs.copy(
-      this.templatePath('src/redux/Auth/reducer.js'),
-      this.destinationPath('src/redux/Auth/reducer.js'),
-      {}
-    );
-    this.fs.copy(
-      this.templatePath('src/services'),
-      this.destinationPath('src/services'),
-      {}
-    );
-    this.fs.copy(
-      this.templatePath('src/utils'),
-      this.destinationPath('src/utils'),
-      {}
-    );
-    this.fs.copy(
-      this.templatePath('src/utils/colors.js'),
-      this.destinationPath('src/utils/colors.js'),
-      {}
-    );
-    this.fs.copy(
-      this.templatePath('src/utils/array.js'),
-      this.destinationPath('src/utils/array.js'),
-      {}
-    );
-    this.fs.copy(
-      this.templatePath('eslintrc.js'),
-      this.destinationPath('.eslintrc.js'),
-      {}
-    );
-    this.fs.copyTpl(
-      this.templatePath('public/_index.html'),
-      this.destinationPath('public/index.html'),
-      { projectName: this.projectName }
-    );
-    this.fs.copy(
-      this.templatePath('src/app/screens/Dashboard/index.js'),
-      this.destinationPath('src/app/screens/Dashboard/index.js'),
-      {}
-    );
-    this.fs.copy(
-      this.templatePath('src/app/screens/Dashboard/screens/Home/index.js'),
-      this.destinationPath('src/app/screens/Dashboard/screens/Home/index.js'),
-      {}
-    );
-    this.fs.copy(
-      this.templatePath('src/services/AuthServices.js'),
-      this.destinationPath('src/services/AuthServices.js'),
-      {}
-    );
-    this.fs.copy(
-      this.templatePath('src/services/LocalStorageService.js'),
-      this.destinationPath('src/services/LocalStorageService.js'),
-      {}
-    );
+  configuring() {
+    return Promise.resolve()
+      .then(() => installCRA.bind(this)())
+      .then(() => runCRA.bind(this)())
+      .then(() => installDependencies.bind(this)());
+
+    //TODO once this feature is in development we have to create a new generator in yeoman with this
```

Comment:
Should we uncomment this? Because if we do so, then these changes should be pushed immediately in order to avoid installing an old version of the bootstrap.